### DISCUSSION TITLE
Is it possible to make the folders clickable in the asset manager?

### DISCUSSION THREAD

**Discussion by carolineboeykens** [craftcms/cms#15238] [Ideas] [answered]
It is more intuitive to click on the folder in the asset manager instead of the title? I think we expect this behaviour because Windows and Apple computers also work like this.

![image](https://github.com/craftcms/cms/assets/42932673/6a0828b3-f015-4758-85a8-5bac2266d3da)

**Comment by brandonkelly** [comment, marked as answer]
Just [added support](https://github.com/craftcms/cms/compare/b90d063fe4ae16d6ab629a32cfd24f45b09882a6...a2ad3b31bb8c53965ff1354c3afa08fe07c49b02) for double-clicking on folders to navigate to them, for Craft 4.11 and 5.3 🎉 

I had originally decided against this because I was concerned about inconsistent behavior between asset indexes/selection modals, and folder selection modals (which are shown when moving a folder, to choose the destination folder). Generally double-clicking is used as a shortcut to choose an element, rather than selecting it and pressing “Select”. So by convention, folder selection modals also (currently) allow you to double-click on them to choose them. Which meant double-clicking _couldn’t_ be used as a means of navigation for folders.

I just checked a folder selection dialog within macOS (specifically, choosing a destination folder that a new repo should be cloned into, within the Tower app), and noticed that they’ve avoided the inconsistency by just not supporting double-click-to-select within folder dialogs. Double-clicking on folders within the dialog will navigate into them, just like a file selection dialog, and the only way to actually choose the current folder is to select it and press the “Open” button. So I followed suit with that, and dropped double-click-to-select support from folder selection modals. Problem solved.

**Reply by brandonkelly** [reply]
Craft 5.3.0-beta.1 is out with that change 🎉

We’re targeting August 6 for the GA release. If you’d like to test ahead of time, change your `craftcms/cms` requirement in composer.json to:

```json
"craftcms/cms": "^5.3.0-beta.1",
```

and then run `composer update`.
